.github/workflows/release.yml

```diff
@@ -9,41 +9,10 @@ on:
 
 
 jobs:
-  release:
-    name: Release new version
-    environment: production
-    runs-on: ubuntu-latest
-
-    if: github.ref_protected == true
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          persist-credentials: false
-          fetch-depth: 0
-
-      - name: Install node
-        uses: actions/setup-node@v3
-        with:
-          node-version-file: '.node-version'
-
-      - name: Install Node packages
-        run: yarn install
-      - name: Lint
-        run: yarn lint
-      - name: Build
-        run: yarn build
-      - name: Tests
-        run: yarn test
-      - name: Type check
-        run: yarn test:dts
-      - name: Semantic Release
-        uses: cycjimmy/semantic-release-action@91ab76a4a393a8d0c4739e9aea1818b56bc953ea
-        with:
-          extra_plugins: |
-            @semantic-release/exec@6.0.3
-            conventional-changelog-conventionalcommits@5.0.0
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
-          HUSKY: 0
+  build-and-release:
+    name: 'Build project, run CI checks and publish new release'
+    uses: fingerprintjs/dx-team-toolkit/.github/workflows/release-typescript-project.yml@v1
+    with:
```

Comment:
It seems to me that followings are missing:
- `nodeVersion` parameter with value read from `.node-version` file
- `yarn build` step (the workflow [looks like](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/release-typescript-project.yml#L76) it downloads `dist` folder without building the project)
- `yarn test` step

`yarn test:dts` is also missing but we are going to remove it anyway so it doesn't matter.

Reply:
[release-typescript-project](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/release-typescript-project.yml) workflow uses [build-typescript-project](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/build-typescript-project.yml) workflow under the hood. So it runs all mentioned commands.

During ecosystem care I expect to start using also [build-typescript-project](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/build-typescript-project.yml) in the [build.yml](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/blob/rc/.github/workflows/build.yml) for this repo.

According to `nodeVersion` I just decided to use default (`'lts/*'`). But we can change it if you have any doubts.

Reply:
Yeah I didn't realize that there was a build step. Looks like `yarn build` and `yarn test` are covered by that step, and I think it is fine to stop using `.node-version` file. I will take a note to remove this file from the project then.